applications/NXreftof.nxdl.xml

```diff
@@ -27,6 +27,21 @@
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
     xsi:schemaLocation="http://definition.nexusformat.org/nxdl/3.1 ../nxdl.xsd"
   >
+  	<symbols>
+      <doc>
+	    The symbol(s) listed here will be used below to coordinate datasets
+		with the same shape.
+      </doc>
+      <symbol name="xSize">
```

Comment:
Why not be consistent with other definitions?  nX (here, nY in next symbol)

Reply:
after a discussion with @benajamin @yayahjb @PeterC-DLS it was decided to leave each definitions names as they were with the new formatting and this idea of standardizing symbol names would be taken up in the future by the NIAC